### PR TITLE
[Windows] Add WiX to VS on Windows-2022

### PR DESCRIPTION
**Pull request by V-Zabayrachny**
# Description
This PR adds extension "WiX" to Visual Studio 2022 on Windows-2022

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/toolsets/toolset-2022.json [hunk 1]
```diff
@@ -243,7 +243,8 @@
             "wasm.tools"
         ],
         "vsix": [
-            "VisualStudioClient.MicrosoftVisualStudio2022InstallerProjects"
+            "VisualStudioClient.MicrosoftVisualStudio2022InstallerProjects",
+            "WixToolset.WixToolsetVisualStudio2022Extension"
```
Comment (miketimofeev) [inline images/win/toolsets/toolset-2022.json:247]: Let's do the same for the VS2019 and VS2017 and get rid of these lines then
https://github.com/actions/virtual-environments/blob/37d8c26d314e6799cb74ec8e378d05cbbaaa3b1a/images/win/scripts/Installers/Install-Wix.ps1#L8-L25

Comment (miketimofeev) [reply images/win/toolsets/toolset-2022.json:247]: And these tests should also be removed
https://github.com/actions/virtual-environments/blob/37d8c26d314e6799cb74ec8e378d05cbbaaa3b1a/images/win/scripts/Tests/Wix.Tests.ps1#L12-L22

Comment (V-Zabayrachny) [reply images/win/toolsets/toolset-2022.json:247]: Ok, I made it.

**Review by miketimofeev — APPROVED** [review]

**Pull request merged**